File: c/iniciante/BEE1005.c
#include <stdio.h>

int main() {

  double a;
  double b;

  scanf("%lf %lf", &a, &b);

  a = a * 3.5;
  b = b * 7.5;

  double media = (a + b) / 11;

  printf("MEDIA = %.5lf\n", media);

  return 0;
}

File: go/iniciante/BEE1005.go
package main

import (
	"fmt"
)

func main() {
	var a float64
	fmt.Scan(&a)

	var b float64
	fmt.Scan(&b)

	var media float64 = ((a * 3.5) + (b * 7.5)) / 11.0

	fmt.Printf("MEDIA = %.5f\n", media)

}